src/modules/comment.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-confusing-arrow */
```

Comment:
Please do not disable linter checks in every other module. It would be nice if you can fix all errors introduced by linter checks.